cds/config.py

```diff
@@ -56,39 +56,39 @@ def _(x):
 
 # RabbitMQ.
 BROKER_URL = os.environ.get(
-    "APP_BROKER_URL",
-    "redis://localhost:6379/0")
+    'APP_BROKER_URL',
+    'redis://localhost:6379/0')
 # Celery results.
 CELERY_RESULT_BACKEND = os.environ.get(
-    "APP_CACHE_REDIS_URL",
-    "redis://localhost:6379/1")
+    'APP_CACHE_REDIS_URL',
+    'redis://localhost:6379/1')
 # Celery accepted content types.
 CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
 
 ###############################################################################
 # Cache
 ###############################################################################
 
-CACHE_KEY_PREFIX = "cache::"
+CACHE_KEY_PREFIX = 'cache::'
 CACHE_REDIS_URL = os.environ.get(
-    "APP_CACHE_REDIS_URL",
-    "redis://localhost:6379/0")
-CACHE_TYPE = "redis"
+    'APP_CACHE_REDIS_URL',
+    'redis://localhost:6379/0')
+CACHE_TYPE = 'redis'
 
 ###############################################################################
 # Database
 ###############################################################################
 
 SQLALCHEMY_DATABASE_URI = os.environ.get(
-    "SQLALCHEMY_DATABASE_URI",
-    "postgresql+psycopg2://localhost/cds")
+    'SQLALCHEMY_DATABASE_URI',
+    'postgresql+psycopg2://localhost/cds')
 SQLALCHEMY_ECHO = False
 SQLALCHEMY_TRACK_MODIFICATIONS = True
 
 ###############################################################################
 # Debugbar
 ###############################################################################
-
+DEBUG = True
```

Comment:
we need to remember to update puppet to set it to `False` in the server if we keep it here like that